flutter_common/lib/data/results/benchmark_result.dart

```diff
@@ -3,6 +3,30 @@ import 'backend_settings.dart';
 import 'dataset_info.dart';
 import 'loadgen_scenario.dart';
 
+class AccuracyData {
+  static const String _tagValue = 'value';
+  static const String _tagString = 'string';
+
+  final double value;
+  final String string;
+
+  AccuracyData({
+    required this.value,
```

Comment:
If the accuracy value is normalized, should we check if the value is in the normalized range and raise an exception if not?

Reply:
I don't think we should check such things in the data class.
Requirement for normalization is a weak requirement. We need it to make value loop pretty in the UI but it doesn't really matter otherwise. From data class standpoint there is no difference.

Also I don't see how we would handle this exception. I think crashing the app because there is an error related to accuracy calculation is a very bad idea.

Maybe we could check that result doesn't exceed the upper bound, and if it does then mark result invalid (red). I'll add this check.

Reply:
> Maybe we could check that result doesn't exceed the upper bound, and if it does then mark result invalid (red). I'll add this check.

Done